CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 * Fix adding a gender type
 * Fix update activity with emotions
 * Fix bad return with wrong credential on oauth/login api
+* Fix aged initial setup instructions
```

Comment:
You don't need to update CHANGELOG.md anymore
```suggestion
```

Reply:
@asbiin why is this? Is this auto-generated in some way?

This was still part of the contributor instructions. I can remove if it's not valid anymore.